### ISSUE TITLE
Accessibility: Using the Tab key to operate the menu only goes up to one sub-level

### ISSUE THREAD

**Issue by WM-Loose** [templaza/astroid-framework#988]
Hello Sonny, hello colleagues,

we have complaints that the menu with the tab keys only works up to one sub-level. With sub-sub, you cannot jump further with the tab. This is a huge problem for accessibility. Click instead of hover is of course set in the template options. If you now have 2 sub-levels, you can only get to the first with tab and enter. From the second sub-level onwards, you can only move forward with the mouse.

Can this be adjusted somehow?

Regards,
Dirk

**Comment by sonvnn** [comment]
I will check and response asap

**Comment by sonvnn** [comment]
@WM-Loose I fixed this issue on https://github.com/templaza/astroid-framework/commit/eb64ce3b44568435b74b015bcfb7164c4b846f01

Please help me download v3-branch and test it.

Best Regards,
Sonny

**Comment by WM-Loose** [comment]
Now it works perfect. 
Very cool job Sonny.
